### PR TITLE
Update and upgrade ventilator allocation page

### PR DESCRIPTION
**Pull request by adelarue**
Two changes:
1) Updated ventilator allocation to use 4/15 data
2) Upgraded intro to ventilator page for larger scope

Once this PR is merged, the last step to add will bee the video link, once we have figured out how to have it securely.

### CODE REVIEW

File: ventilators/allocations.py [hunk 1]
```diff
@@ -19,25 +19,148 @@
...
+                    xs=12,
+                    sm=6,
+                    md=6,
+                    lg=4,
+                    xl=4,
+                    style={"paddingBottom": "20px"},
+                ),
+                dbc.Col(
+                    dbc.Card(
+                        [
```
Comment (galitLukin) [inline ventilators/allocations.py:73]: In desktop, this card has a white space on the bottom while the others dont. In mobile it looks fine. If you dont see this space, let me know and ill take a screenshot.

File: ventilators/allocations.py [hunk 2]
```diff
@@ -19,25 +19,148 @@
...
+                    sm=6,
+                    md=6,
+                    lg=4,
+                    xl=4,
+                    style={"paddingBottom": "20px"},
+                ),
+            ],
+            justify="around",
+        ),
+        # can comment out until we figure out the secure connection, this is a placeholder
```
Comment (galitLukin) [inline ventilators/allocations.py:127]: not for this pr but this page is already really long imo and all of its content is really valuable. I wont remove anything and def. not this video. I think we should consider splitting the content into two pages

File: ventilators/allocations.py (hunk 1 above)
Comment (hwiberg) [reply ventilators/allocations.py:73]: I see the same thing as Galit.

File: ventilators/allocations.py (hunk 2 above)
Comment (hwiberg) [reply ventilators/allocations.py:127]: We could put the baseline graphs as a separate page? I agree it's a bit long and the stuff we want to highlight is at the bottom but it doesn't make sense to move the baseline below.. we can talk about this more but nothing for the PR!

Comment (hwiberg) [reply ventilators/allocations.py:127]: We could put the baseline graphs as a separate page? I agree it's a bit long and the stuff we want to highlight is at the bottom but it doesn't make sense to move the baseline below.. we can talk about this more but nothing for the PR!

File: ventilators/allocations.py (hunk 1 above)
Comment (adelarue) [reply ventilators/allocations.py:74]: Y'all have eagle eyes! It should be fixed now.

File: ventilators/allocations.py (hunk 2 above)
Comment (adelarue) [reply ventilators/allocations.py:130]: Agreed - the page is too long. Once it is approved and we have the video as an embedding and not just a link, I think we can split it the following way: 
1. an overview page with the motivation, three cards (hospitals, states, and countries), and embedded video
2. an interactive page following on the state allocation

**Pull request merged**